conf/core.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE if not exists core_result (link_id integer, output text, parsed dat
 CREATE TABLE if not exists core_ability (id text, technique integer, name text, test text, description text, cleanup text, executor, UNIQUE (id, executor) ON CONFLICT IGNORE);
 CREATE TABLE if not exists core_group (id integer primary key AUTOINCREMENT, name text, UNIQUE(name));
 CREATE TABLE if not exists core_group_map (group_id integer, agent_id integer, UNIQUE(group_id, agent_id));
-CREATE TABLE if not exists core_adversary (id integer primary key AUTOINCREMENT, name text, description text, UNIQUE (name));
+CREATE TABLE if not exists core_adversary (id integer primary key AUTOINCREMENT, name text, description text, locked integer, UNIQUE (name));
```

Comment:
make sure you also add:
ALTER TABLE core_adversary ADD COLUMN locked integer;
to the bottom. this will add the column if the table already exists.

Reply:
Added the appropriate ALTER TABLE commands to update existing databases.